src/lang_utils/error_codes.ml

```diff
@@ -198,6 +198,6 @@ let error_codes : (string * string option) list =
     "M0192", None; (* Object/Actor/Module body type mismatch *)
     "M0193", None; (* Can't declare actor class to have `async*` result *)
     "M0194", Some([%blob "lang_utils/error_codes/M0194.md"]); (* Unused identifier warning *)
-    "M0195", None; (* warn that `async` demand is getting passed behind the scenes *)
-    "M0196", None; (* attempt to pass `async` demand not accepted by callee *)
+    "M0195", None; (* warn that `system` demand is getting passed behind the scenes *)
+    "M0196", None; (* attempt to pass `system` demand which is not accepted by callee *)
```

Comment:
Might be good to provide some .md files now, to avoid answering the same question on the forum over and over again....

Reply:
Done in da6a7c4a23a9c3a01abe365f0d6bc71d73b3aff2.